src/scripts/experience/explore-mode.js

```diff
@@ -302,13 +313,19 @@ define(['pixi', 'experience/mode', 'experience/issue'], function (PIXI, Mode, Is
             issue.exploreMouseOver = this._mouseOverIssue.bind(this);
             issue.exploreMouseOut = this._mouseOutIssue.bind(this);
             issue.exploreTap = this._tapIssue.bind(this);
+            issue.exploreFocus = this._focusIssue.bind(this);
```

Comment:
I believe the bound handlers are stored on the issue so that they can be removed within `_onStartHide`, as you need a reference to the handler to remove it from the signal. I started to refactor this to be cleaner, but then realized that I'd need to do that for every mode and handler to be consistent, which seemed like a rather large and out-of-scope change, so I went with the verbose method here. For now. :P